vfs/file.go

```diff
@@ -784,3 +787,46 @@ func (f *File) Truncate(size int64) (err error) {
 	}
 	return nil
 }
+
+// Note: If we end up adding many different xattrs, rewrite this function into
+// something more extensible.
+func (f *File) Getxattr(name string) (value []byte, err error) {
+	obj := f.getObject()
+
+	if name == "user.mime_type" {
+		return []byte(fs.MimeType(context.TODO(), obj)), nil
```

Comment:
In the mount2 backend it does give us Context objects that could be passed up. I see however that none of these other methods accept a Context, so I didn't add it to this one.

Do you think it's worth adding a Context argument for Getattr/Listattr?

Reply:
Opted to pass the context object through.